src/lib/containers/widgets/query-filter/QueryFilter.tsx

```diff
@@ -138,22 +139,40 @@ export const applyChangesToRangeColumn = (
 export const QueryFilter: React.FunctionComponent<QueryFilterProps> = ({
   facetsToFilter,
 }): JSX.Element => {
+  const [facets, setFacets] = React.useState<FacetColumnResult[]>([])
+  const [facetFilter, setFacetFilter] = React.useState(facetsToFilter ?? [])
```

Comment:
Does this variable name change make it more clear?
```suggestion
  const [facetFiltersShown, setFacetFiltersShown] = React.useState<string[]>(facetsToFilter ?? [])
```